scripts/publish-npm.ts

```diff
@@ -268,23 +287,19 @@ async function main() {
   // 3. Interactively on the command line.
   let packages: string[];
   if (args.packages.length > 0) {
-    // Get packages to publish from args
-    const errorMessages: string[] = [];
+    // Get packages to publish from the 'packages' arg
     // Filter from the set of all packages to make sure they end up
     // in topological order.
     const allPackages = getPackages(PUBLISHABLE_RELEASE_UNITS);
-    const toPublish = new Set(args.packages);
-    packages = allPackages.filter(pkg => {
-      if (!toPublish.has(pkg)) {
-        errorMessages.push(`Package ${pkg} is not a tfjs package.`);
-        return false;
-      }
-      return true;
-    })
-
-    if (errorMessages.length > 0) {
-      throw new Error(errorMessages.join('\n') +
-        `Supported packages are:\n${[...ALL_PACKAGES].join('\n')}`);
+    const requestedPackages = new Set(args.packages);
+    packages = allPackages.filter(pkg => requestedPackages.has(pkg));
+
+    // Check if there are any unsupported packages requested by the user
+    const unsupportedPackages = setDifference(requestedPackages,
+                                              new Set(packages));
+    if (unsupportedPackages.size > 0) {
+      throw new Error(`Can not publish ${[...unsupportedPackages]}. `
+              + `Supported packages are:\n${[...ALL_PACKAGES].join('\n')}`);
```

Comment:
This was improperly parsing the CLI args. Now, it parses all remaining arguments as packages to publish.